### ISSUE TITLE
Unable to display a single circumflex character (^) in a table cell while enableExtendedTableSyntax is true

### ISSUE THREAD

**Issue by L483** [shd101wyy/markdown-preview-enhanced#1573]
Thank you for your great VS Code extension, I really enjoy working with it. Today I ran into the problem that I was not able to display a single circumflex `^`  in a table cell. Of course, this is due to the enabled extended table syntax and it works as it should. The only problem is that it works too good, e.g. `\^` leads also to a rowspan.

I could not figure out any way of escaping the character so that only a single circumflex is displayed in the table cell. As a solution, I would suggest introducing an escape mechanism via a backslash - as it is already the case with many other special characters.

As this is a very _niche_ request, I could not find out whether this would violate any markdown syntax standard/ flavor.

**Comment by Ghost4Man** [comment]
What's even weirder, all of these will also cause a merged cell (same with `>` instead of `^`):

- `&#94;`
- `<p>^</p>`
- `[^](somefile.pdf)`
- `[&#94;](somefile.pdf)`
- `<input type="text">^</input>`

![screenshot](https://user-images.githubusercontent.com/8436102/118402024-5f7fd800-b668-11eb-9a25-6a13cbbe026f.png)

Which is certainly very weird and confusing behavior. I would expect only `^` and `>` _in the input markdown_ (and not all the things that _render in the output_ as `^` or `>`) to merge the cells.

**Comment by L483** [comment]
Exactly, I also tried your first two options before opening this issue, as they were the only things that came to my mind. In hindsight, I admit that my issue description was not precise enough, so thank you for your comment.
